spec/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ Required ] I think it would be nice if you add a test for the `recent_posts` method in `user.rb`, according to the project requirements you need to add unit specs for all of your models' methods and validations. Please add a unit test for this method.

## Project requirements
![Screenshot from 2023-09-22 01-23-30](https://github.com/grauJavier/blog-app-ruby/assets/16392046/f264a43d-8867-44cd-951a-c7af3367e38f)